dkist/net/tests/test_attr_walker.py

```diff
@@ -53,6 +97,44 @@ def test_walker_single(all_attrs_classes, api_param_names):
     assert not set(api_param_names[all_attrs_classes]).difference(params[0].keys())
 
 
+def test_boundingbox_containing(boundingbox_param):
+    bb_query = da.BoundingBox(bottom_left= boundingbox_param[0], top_right= boundingbox_param[1],
+                       width= boundingbox_param[2], height=boundingbox_param[3], search="containing")
+
+    out = walker.create(bb_query)
+    assert len(out) == 1
+    assert all([isinstance(a, dict) for a in out])
+
+    #can't verify exact coordinates, they change a bit
```

Comment:
we need at least one test case which checks we are formatting the coordinates correctly, as the dataset search api expects them in a given form.